Blish HUD/Strings/GameServices/OverlayService.resx

```diff
@@ -223,4 +223,10 @@ It might not be installed.</value>
   <data name="Setting_DynamicHUDMenuBar_DisplayName" xml:space="preserve">
     <value>Dynamic HUD Menu Bar</value>
   </data>
+  <data name="Setting_DebugLogging_Description" xml:space="preserve">
+    <value>If enabled Blish HUD will create more verbose logfiles</value>
+  </data>
+  <data name="Setting_DebugLogging_DisplayName" xml:space="preserve">
+    <value>Enable debug logging</value>
```

Comment:
Let's revise this to `Enable Debug Logging` as the rest of our settings use title caps.